### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by silviatofana**
In this PR I did the following:
Create a table named vets
Create a join table called specializations and visits
Insert the data for vets, specializations, and visits
Write queries
Attached here are the screenshots :

image

image

image

image

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+  CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
+);
+-- create visits table
+CREATE TABLE visits(
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  animals_id INT,
+  vets_id INT,
+  date_of_visit DATE,
+  CONSTRAINT fk_visit_animals FOREIGN KEY(animals_id) REFERENCES animals(id),
+  CONSTRAINT fk_visits_vet FOREIGN KEY(vets_id) REFERENCES vets(id)
```
Comment (BenMukebo) [inline schema.sql]: - [OPTIONAL ] Kindly, According to the naming convention, the foreign key must always be a singular name example:
```
  animal_id INT,
  vet_id INT,
```

File: data.sql [hunk 2]
```diff
@@ -1,32 +1,442 @@
...
+SELECT * FROM animals;
+
+-- insert data into vets table
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Vet William Tatcher', 45, 'Apr 23 2000'),
+('Vet Maisy Smith', 26 ,'Jan 17 2019'),
+('Vet Stephanie Mendez', 64, 'May 04 1981'),
+('Vet Jack Harkness', 38,'Jun 08 2008');
+
```
Comment (BenMukebo) [inline data.sql]: - [ ] You were instructed to add the following data for `vets` table 👇🏻  but the name is different, 

  - Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
  - Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
  - Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
  - Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
- [ ] Kindly remove the `Vet` in front of your names, your code should be like this:

```sql
   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Maisy Smith', 26, '2019-01-17');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Stephanie Mendez', 64, '1981-05-04');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Jack Harknes', 38, '2008-06-08');
```

File: data.sql [hunk 3]
```diff
@@ -1,32 +1,442 @@
...
+            FROM species
+            WHERE name = 'Digimon'
+        ),
+        (
+            SELECT id
+            FROM vets
+            WHERE name = 'Vet Jack Harkness'
+        )
+        );
+
```
Comment (BenMukebo) [inline data.sql]: - [ ] Kindly don't forget to modify the 'Vet Stephanie Mendez' and 'Vet Stephanie Mendez' names after removing the `vet`. Your code might look like this:

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @silviatofana,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @silviatofana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
thanks

**Pull request merged**